### ISSUE TITLE
Website Accessibility-WCAG 1.1.1: Ensures <img> elements have alternate text or a role of none or presentation (.navbar-logo)

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14491]
#### Issue
`Ensures <img> elements have alternate text or a role of none or presentation` ([image-alt](https://accessibilityinsights.io/info-examples/web/image-alt))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
.navbar-logo
#### Snippet
`<img class="navbar-logo" src="/images/bazel-navbar.svg">`
#### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Microsoft Edge version 96.0.1054.62

**Comment by fweikert** [comment]
We've migrated to a new site at https://bazel.build that fixes several accessibility issues.
